### DISCUSSION TITLE
alternative keybinding for neovim integration

### DISCUSSION THREAD

**Discussion by tom-pang** [community/community#6899] [Copilot Conversations] [answered]
It'd be nice to bind a different key rather than tab, because I already use that for LSP based completion (which I see as an alternative, to use alongside copilot). But I don't see anything in the docs about doing that.

**Comment by tpope** [comment, marked as answer]
You're in luck, this is supported (but not yet documented).

```vim
inoremap <silent><expr> <C-]> copilot#Accept("")
let g:copilot_no_tab_map = 1
```

The part in parentheses is the fallback when there's nothing to accept. An empty string does nothing.

**Reply by Lite5h4dow** [reply]
is there a way to have it fallback to requesting a suggestion instead of nothing or a key input
